site/src/components/Table/Table.test.tsx

```diff
@@ -1,81 +0,0 @@
-import { screen } from "@testing-library/react"
-import { render } from "../../testHelpers/renderHelpers"
-import { Column, Table } from "./Table"
-
-interface TestData {
```

Comment:
Why did we delete the test file?

Reply:
We are not using those components anymore. They were wrappers around the material UI ones that we used in V1 that are no longer necessary.